SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -88,6 +88,26 @@ void UpdateChangelistHistory(TUniquePtr<FRepState>& RepState)
 	SendingRepState->HistoryStart = SendingRepState->HistoryStart % MaxSendingChangeHistory;
 	SendingRepState->HistoryEnd = SendingRepState->HistoryStart + NewHistoryCount;
 }
+
+void ForceReplicateOnActorHierarchy(USpatialNetDriver* NetDriver, const AActor* RootActor, const AActor* OriginalActor)
```

Comment:
Shouldn't need to pass in the NetDriver here? SpatialActorChannel already has a reference to it?

Reply:
Also would prefer name changes of `RootActor` -> `HierarchyActor` and `OriginalActor` to `OriginalActorBeingReplicated`